reframe/frontend/statistics.py

```diff
@@ -33,7 +33,8 @@ def tasks(self, run=-1):
             raise errors.StatisticsError('no such run: %s' % run) from None
 
     def failures(self, run=-1):
-        return [t for t in self.tasks(run) if t.failed]
+        return [t for t in self.tasks(run) if (t.failed and
+                                               not t.aborted)]
```

Comment:
Would it make sense that `t.failed` was true only if the test has failed and be aborted?